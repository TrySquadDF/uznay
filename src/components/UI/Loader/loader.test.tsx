import * as React from "react";
import { render } from "../../../utils/test-utils";
import { Loader } from ".";

describe("Loader Test State", async () => {
  it("Загрузчик видно", () => {
    const { getByTestId } = render(<Loader />);
    expect(getByTestId("loader")).toBeInTheDocument();
  });

  it("Загрзучик такой же Зеленый как и был", () => {
    const { getByTestId } = render(<Loader />);
    expect(getByTestId("loader")).toMatchSnapshot();
  });
});
