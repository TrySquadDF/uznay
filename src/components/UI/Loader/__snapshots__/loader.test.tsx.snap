// Vitest Snapshot v1

exports[`Loader Test State > Загрзучик такой же Зеленый как и был 1`] = `
<div
  class="c-cyZEAk"
  data-testid="loader"
>
  <div
    class="c-jTKaZN"
  >
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
    <span
      class="c-cvlFAN"
    />
  </div>
</div>
`;
